monitoring/base/rhods-rules.yaml

```diff
@@ -16,5 +16,9 @@ spec:
       expr: count(kube_pod_container_status_ready{namespace="rhods-notebooks",pod=~"jupyterhub-nb.*"}==1)
     - record: cluster:usage:consumption:rhods:active_users
       expr: count(count_over_time(kube_pod_container_status_ready{namespace="rhods-notebooks",pod=~"jupyterhub-nb.*"}[1h]))
+    - record: cluster:usage:consumption:rhods:cpu_requests_runtime
```

Comment:
Based on where you are inserting this, aren't you "moving" the `instance: jupyter-notebooks` label from `cluster:usage:consumption:rhods:active_users` to `cluster:usage:consumption:rhods:cpu_limits_runtime` ?

Reply:
Could you elaborate on what "moving" the `instance: jupyter-notebooks` label means?
I assumed that all these rules apply to the jupyterhub instance and hence I just added these two under rules, while keeping the structure between rules and labels as is.

Reply:
Done, updated to have the label moved up to the `cluster:usage:consumption:rhods:active_users` rule based on discussion with @anishasthana.